versioned/tests/src/main/java/org/projectnessie/versioned/StringStoreWorker.java

```diff
@@ -89,24 +89,22 @@ public String getId(String contents) {
 
   @Override
   public Byte getPayload(String contents) {
-    return getValueSerializer().getPayload(contents);
+    return 0;
   }
 
   @Override
   public TestEnum getType(Byte payload) {
-    return getValueSerializer().getType(payload);
+    if (payload == null) {
+      return StringStoreWorker.TestEnum.NULL;
+    }
+    return payload > 60 ? StringStoreWorker.TestEnum.YES : StringStoreWorker.TestEnum.NO;
```

Comment:
why 60 and not 42 ;-)

Reply:
IDK - was like that before. It's a test-only thing though ;)

Reply:
I'd have DEFINITELY chosen 42 :P